spec/factories/form526_submissions.rb

```diff
@@ -3,9 +3,9 @@
 FactoryBot.define do
   factory :form526_submission do
     user_uuid SecureRandom.uuid
-    saved_claim_id '123'
+    saved_claim { create(:va526ez) }
     submitted_claim_id nil
-    auth_headers_json ''
+    auth_headers_json 'null'
```

Comment:
@kreek @SMLuthi in the new version of `attr_encrypted`, blank string for the `auth_headers_json` field violates the not-null db constraint, do we want to require a non-blank value for this field or remove the not-null db constrait?

Reply:
@lihanli in the app flow auth headers should always be available so we could require a non-blank string.